src/client/FlagsClient.ts

```diff
@@ -49,6 +50,13 @@ class FlagsClient {
     return this.flags.filter((flag) => flag.name === flagName)[0];
   }
 
+  /**
+   * Get the user ID using the hook passed into the config
+   */
+  public getUserId(): string | undefined {
+    if(this.config.userId) { return this.config.userId}
```

Comment:
if statement can be removed. If this will be moved to the context the entire method can be removed as well